### DISCUSSION TITLE
heptane pool fire, validation

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#13107] [Q&A]
Hello,

I have run the validation example code simulating a heptane pool fire using version 6.7.0. (without any modifications)
However, the result values differed from those in the FDS validation. I would appreciate any advice on this matter. 
Thank you.

Validation
![image](https://github.com/firemodels/fds/assets/166811458/98d758d8-8edc-4981-8731-1f7792c23315)
my result
![image](https://github.com/firemodels/fds/assets/166811458/17973b2a-1ddf-4a2d-88ca-cb910c2d6c80)

[heptane_1_m2.txt](https://github.com/user-attachments/files/16046789/heptane_1_m2.txt)

**Comment by TristanHehnen** [comment]
Please look at the title of the plot you found in the validation guide, it states that an FDS version was used that is 1936 changes after the 6.6.0 tag: 
![grafik](https://github.com/firemodels/fds/assets/34650649/7a5dc1d1-11b4-4775-b470-0b9bef7f695c)


According to your post, you are using FDS 6.7.0.
As we discussed previously (#13090 ), different FDS versions may lead to different results. 

Unfortunately, the validation guide can contain inconsistencies with respect to the FDS versions used for a given case. Meaning, if you look in the validation guide provided with an installer, it may contain cases which were run with earlier versions, see table "Table 16.2: Validation Git statistics for all data sets" as well as the plot titles. You can have a look at newer versions of the validation guide ([assets](https://github.com/firemodels/fds/releases)), to see if it was run at a later point with the desired version. 
Another option would be, to look for which version a case was run in the validation guide and then run your own simulations with this specific version.

You can further look at the [file history on git](https://github.com/firemodels/fds/commits/master/Validation/Pool_Fires/FDS_Input_Files/VTT_heptane_1_m2.fds), to make sure you are using the correct input file:
![grafik](https://github.com/firemodels/fds/assets/34650649/24d22d47-9cbd-4f8e-87cf-c6c5682af86b)

There are also some slight differences between your file and the most recent version on git: the validation case uses multiple meshes and also has a floor. Your case has an `OPEN` boundary at the domain bottom. 
Furthermore, the validation case setup contains multiple `DEVC`, one of which records the `MLRPUA` with some statistics which are missing in your setup. Make sure you are analysing the same quantity as was used for the plot in the validation guide.

You have been asking very similar questions multiple times now. Could you please elaborate a bit more on what you intent to do and what you expect to achieve? 
As it stands now, I do not know what else to tell you other than that different FDS versions lead to different results.

**Reply by MNJOLE** [reply]
Yes, I understand from your previous response that the results of FDS can vary depending on the version. Since the graph I uploaded is from the validation material of version 6.7.0, I assumed that the results would be the same for both versions 6.6.0 and 6.7.0, which might have caused this situation. Thank you for providing additional information regarding validation, especially pointing out the significance of the floor's presence. I will make adjustments accordingly.

The reason I did not install DEVC MLRPUA was that the MLR_FUEL is automatically calculated in the HRR.csv file. Could you please clarify if there is a difference between the MLRPUA value obtained by installing DEVC and the value specified by MLR_FUEL/Area? (I am wondering if I am missing something here.) 

Additionally, I am currently working on analyzing a model where liquid fuel (solid fuel) undergoes pyrolysis. As I was reviewing the code the FDS provided, I encountered some confusion. (The fact that the shape of the HRR/burning rate itself changed was quite surprising, but I understand it now.) I will extend my research using the methods you suggested. Thank you for your thoughtful advice and answers.

**Comment by drjfloyd** [comment]
You are not comparing the same two input files. The 2018 repo had only the heptane_1_m input file. The m2 file wasn't added until August of 2018, two months after the release of 6.7.0 in June 2018. 

<img width="246" alt="image" src="https://github.com/firemodels/fds/assets/12799217/016f5679-c688-4389-b67a-18229f4ec270">

Yes, MLR_FUEL in the hrr file should be equivalent to integrating the MLRPUA over the burner and the dividing by the area (which is done using CONVERSION_FACTOR).

The validation suite takes a very long time to run. It is some 3000 cases now. For each major and minor release we strive to ensure that all validation cases have been run recently. The specific git hash in the plot is for a working version between 6.6.0 and 6.70 dated 5/10/2018.  6.7.0 was June 18, 2018 and 6.6.0 was Oct. 31, 2017. So that heptane case was run within a month of release. Looking at the log for the source directory, the commits between May 10 and June 18 were mostly related to GEOM, pyro3d, and outputs.
